src/search.cpp

```diff
@@ -276,21 +280,31 @@ namespace search {
             }
         }
 
-        fs::path calculate_prefix(const fs::path & path) {
+        fs::path calculate_prefix(const fs::path & path, std::string_view name) {
             // TODO: Windows
-            // TODO: /cps/<name-like>
+            // TODO: Mac
+
             std::vector<std::string> split = utils::split(std::string{path}, "/");
+            // Sometimes the split() function puts an empty string
```

Comment:
Is empty string different than `.`? What does `PKG_CONFIG_PATH` do?

Reply:
In this case the split function is just a dumb string splitting function, so if the string ends in `/` then the vector will have a `""` at the end.

Reply:
I've updated this comment to try to better describe what's happening.